### ISSUE TITLE
Better error message when vlfeat not found

### ISSUE THREAD

**Issue by AbdealiJK** [menpo/cyvlfeat#15]
Currently if I try to install cyvlfeat without vlfeat installed, I get:
```
cyvlfeat/sift/cysift.c:280:21: fatal error: vl/host.h: No such file or directory
  compilation terminated.
```
This makes sense, because `vl/host.h` cannot be found, as vlfeat is not installed.
But, it would be nicer to mention something like "Could not find `vlfeat` <http://www.vlfeat.org/>. Please install it and then install cyvlfeat"

NOTE: This is again, when I use this with `pip`. The readme says vlfeat is auto installed in conda (I don't normally use conda so have not tried with conda)

You can see the whole error message at https://s3.amazonaws.com/archive.travis-ci.org/jobs/132164694/log.txt

**Comment by patricksnape** [comment]
Yes this is currently expected behaviour as pip installing is not recommended not fully supported at the moment as you must satisfy the vlfeat dependency yourself. This is essentially the same problem as #14.

**Comment by AbdealiJK** [comment]
What I was thinking when creating this issue was that a better error
message should be given. I understand that vlfeat cannot be installed with
pypi.
But something like saying "Hey there, please install vlfeat and try again"
instead of the error message it gives right now

**Comment by AbdealiJK** [comment]
I thought they were separate issues (As this is simply giving a better
error message) and this would be simpler. But if this gets solved by #14
<https://github.com/menpo/cyvlfeat/issues/14>, then please do close this :)

**Comment by patricksnape** [comment]
We could sort this separately but it would certainly be part of #14.
